contracts/services/astroport-lper/src/msg.rs

```diff
@@ -0,0 +1,164 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{ensure, Addr, Decimal, Deps, Uint128};
+use cw_ownable::{cw_ownable_execute, cw_ownable_query};
+
+use crate::error::ServiceError;
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub owner: String,
+    pub processor: String,
+    pub config: ServiceConfig,
+}
+
+pub trait ServiceConfigValidation<T> {
```

Comment:
We have a top level package for this but it´s not compatible with this particular contract because we need to user older dependencies for astroport, so we're kinda used to rewrite stuff

Reply:
we have a top level package for this but unfortunately it's not compatible because I have to lower all our dependencies to work with astroport, so I'm redefining here for this particular case

Reply:
just to clarify, this needs to be implemented because it's going to be used for our workflow or something like that to verify configs, so I'm basically doing the same that the rest of the services are doing

Reply:
The only workaround I can think of is to copy the Astroport API instead of using theirs

Reply:
Like we discussed, refactored this to use all our utils and rewrite Astroport API using cosmwasm 2.0 instead